💎(site) Add New Design to Technical Landing - Body

Vanss472 opened the pull request:
- added interest form to tech detail template
- added get started section to tech detail template

**Please tag each issue this Pull Request resolves**

- Resolves #339 
- Resolves #291

@Snugug my bad here I pushed the Technical content changes to the **content/getting-started** branch directly. Let me know if you need me to reverse it. 

Also about the interest form section. In my opinion, it looks better if the CTA is full width rather than what you recommended.

Screenshot if side-by-side:
![Screen Shot 2022-04-06 at 11 52 10 AM](https://user-images.githubusercontent.com/1170192/162018978-5c0d2b81-03de-45ee-a938-95425f1b27be.png)

vs full width:
![image](https://user-images.githubusercontent.com/1170192/162019571-98ace65f-50c9-4de9-84b1-d7c90b8b931e.png)

Snugug committed:
🐛(build) Fix custom markdown linting (#406)

* Add technical hero schema

* Update theme schema

* Remove company as required in apps schema

* Update title and metadesc min and max lengths

* Fix custom markdown linting

* Fix newfound linting issues

* Fix failing tests

Commit:
remove desc propertie from frontmatter

Snugug reviewed (changes requested):
Thanks for this! I've got a few small comments, but otherwise it looks great!

Commit:
Update site/en/android/index.md

Co-authored-by: Sam Richard <snugug@google.com>

Vanss472 commented on lib/linting/schemas/technical.js:
```diff
@@ -31,14 +31,85 @@ module.exports = {
+            title: {
+              type: 'string',
+              description: 'Title to display in get started section',
+              minLength: 10,
```
set to `maxLength: 500,`

Vanss472 commented on site/sass/layouts/_tech-detail.scss:
```diff
@@ -92,6 +92,60 @@
+    max-width: 60ch;
+    padding: 2rem 2rem 2.5rem;
+
+    p {
+      color: var(--grey-700);
+    }
```
fixed

Vanss472 commented on site/sass/layouts/_tech-detail.scss:
```diff
@@ -92,6 +92,60 @@
+      width: 100%;
+      z-index: 1;
+
+      p {
+        color: var(--grey-700);
+      }
```
fixed

Vanss472 commented on site/_layouts/tech-detail.njk:
```diff
@@ -32,7 +32,32 @@ layout: _wrapper
+          <header class="tech-detail__get-started--heading">
+            <h3 class="type--h4">{{ card.title }}</h3>
+          </header>
+          {{ card.body | markdown | safe }}
```
done.

Commit:
1. Set max length on technical started section and interest for title
> 2. Wrapped started card body in type and a custom class
> 3. Removed p tag and using a custom class instead

Snugug committed:
Merge branch 'main' into content/getting-started

Snugug commented:
Thanks for your work here! I'm calling this _done_! I'm not going to merge yet, as I want to update our other sections with getting started content.

Snugug committed:
Update links and tense

Snugug committed:
Merge branch 'main' into content/getting-started
